ui-src/src/App.js

```diff
@@ -1,78 +1,110 @@
-import React, { Component } from 'react'
-import { Route, Link } from 'react-router-dom'
-import SettingsContainer from './SettingsContainer'
-import FollowContainer from './FollowContainer'
-import NewMeowContainer from './NewMeowContainer'
-import FollowingFeedContainer from './FollowingFeedContainer'
-import UserFeedContainer from './UserFeedContainer'
-import MeowContainer from './MeowContainer'
+import React, { Component } from 'react';
+import { Route, Link } from 'react-router-dom';
+import EditProfileContainer from './EditProfileContainer';
+import FollowContainer from './FollowContainer';
+import FollowingFeedContainer from './FollowingFeedContainer';
+import MeowContainer from './MeowContainer';
+import Modal from './components/Modal';
+import NewMeowContainer from './NewMeowContainer';
+import SettingsContainer from './SettingsContainer';
+import UserFeedContainer from './UserFeedContainer';
 
 class App extends Component {
-  componentWillMount () {
+  componentWillMount() {
     // this fetches the hash which represents the active users userHash
-    this.props.getMyHandle()
-    this.props.getHandles()
-    this.interval = setInterval(this.props.getHandles, 2000)
+    this.props.getMyHandle();
+    this.props.getHandles();
+    this.interval = setInterval(this.props.getHandles, 2000);
   }
 
-  componentDidUpdate (prevProps) {
+  componentDidUpdate(prevProps) {
     // console.log(prevProps.handle, this.props.handle)
     if (!prevProps.handle && this.props.handle) {
-      this.props.getFollow(this.props.handle, 'following')
+      this.props.getFollow(this.props.handle, 'following');
     }
   }
 
-  componentWillUnmount () {
-    if (this.interval) clearInterval(this.interval)
+  componentWillUnmount() {
+    if (this.interval) clearInterval(this.interval);
```

Comment:
Not sure if you set a styleguide that requires semicolons, but JS does not (any more).
(And I personally would prefer not having them but ok :D)